sgl-kernel/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+
+docker run --rm -it \
```

Comment:
An example of a docker build is provided, and it will be automated in follow-up PRs.